HP_41C

```diff
@@ -0,0 +1,3 @@
+01 LBL "HELLO"
+02 "HELLO WORLD"
```

Comment:
Is it possible to change `HELLO WORLD` into `Hello World`?

Reply:
I think no 

![SAVE_20210123_124833](https://user-images.githubusercontent.com/74637789/105571861-60b78d00-5d79-11eb-9cdd-026cfa7d79d5.jpg)